wrapper/src/main/java/software/amazon/jdbc/util/PropertyUtils.java

```diff
@@ -100,7 +103,7 @@ public static void setPropertyOnTarget(
           () ->
               Messages.get(
                   "PropertyUtils.failedToSetPropertyWithReason",
-                  new Object[] {propName, target.getClass(), ex.getCause().getMessage()}));
+                  new Object[] {propName, target.getClass() == null ? Messages.emptyArgs : target.getClass(), ex.getCause() == null ? Messages.emptyArgs : ex.getCause().getMessage()}));
```

Comment:
target == null || target.getClass() == null ?

Reply:
ya, that is simpler.